frontend/components/form/checkbox/FormCheckboxBtn.vue

```diff
@@ -51,3 +52,16 @@ const customColorClass =
     ? `bg-light-${props.customColor}/60 dark:bg-dark-${props.customColor}/10 dark:text-dark-${props.customColor}`
     : "";
 </script>
+
+<style scoped>
```

Comment:
Hey @shubhalgupta 👋 As we use Tailwind for CSS styling, we should be defining colors in the [tailwind.config.ts](https://github.com/activist-org/activist/blob/main/frontend/tailwind.config.ts) file. Do you want me to make the change, or should I?

Reply:
When you define the colors it would be

```json
"light": {
    "hover": "#ffdd00"
}
```

But we'd need to convert it into RGBA as is done in that file :)